### PR TITLE
Small optimization and improved test

### PR DESCRIPTION
**Pull request by masipcat**

### CODE REVIEW

File: jobs/tests/test_api.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ async def test_jobs_basic_operations(db):
     assert await count(db, "jobs.job_queue") == 1
     tasks = await jobs.consume(db, 1)
     assert len(tasks) == 1
+    assert tasks[0]["run_at"] is not None
```
Comment (masipcat) [inline jobs/tests/test_api.py:24]: I think this should work but for some reason it fails. Any idea @jordic

File: jobs/sql/0001_schema.up.sql [hunk 2]
```diff
@@ -326,7 +326,7 @@ create or replace function jobs.consume(topic varchar, num integer)
 BEGIN
     PERFORM jobs.clean_timeout();
     RETURN QUERY WITH tasks AS (
-        SELECT *
+        SELECT id
```
Comment (masipcat) [inline jobs/sql/0001_schema.up.sql:329]: It seems that we only use the field `id` from the CTE to filter the select

File: jobs/tests/test_api.py (hunk 1 above)
Comment (jordic) [reply jobs/tests/test_api.py:24]: It could be, because cte's are run multithreaded.

**Comment by jordic** [comment]
Added partially this PR and fixed the issue with the test on 
https://github.com/vinissimus/jobs/commit/6e15749465f7da44e4dc0ad2f520ea6f7fbb67fe

**Pull request closed without merging**